Modules/_bisectmodule.c

```diff
@@ -16,7 +16,8 @@ module _bisect
 _Py_IDENTIFIER(insert);
 
 static inline Py_ssize_t
-internal_bisect_right(PyObject *list, PyObject *item, Py_ssize_t lo, Py_ssize_t hi)
+internal_bisect_right(PyObject *list, PyObject *item, Py_ssize_t lo, Py_ssize_t hi,
+                      PyObject* key)
```

Comment:
@rhettinger thank you so much for adding `key` function parameter. Just a small typo here: `PyObject* key` -> `PyObject *key`